.circleci/config.yml

```diff
@@ -102,38 +65,9 @@ jobs:
       - run:
           name: Run Lint
           command: |
-            docker create -v /src/charmander/tests/ --name service_tests alpine:3.4 /bin/true
```

Comment:
Compare this diff with what `lint` actually does based on the docker entrypoint: https://github.com/globality-corp/globality-build/blob/master/globality_build/templates/build/python-service/entrypoint.sh#L50